components/raftstore/src/store/fsm/store.rs

```diff
@@ -691,6 +707,37 @@ impl<EK: KvEngine, ER: RaftEngine, T: Transport, C: PdClient> RaftPoller<EK, ER,
             self.poll_ctx.raft_metrics.ready.snapshot - self.previous_metrics.ready.snapshot
         );
     }
+
+    fn flush_ticks(&mut self) {
+        const TICKS: &[PeerTicks] = &[
```

Comment:
How about adding a method to PeerTicks to return all ticks?

Reply:
Ok.